moler/config/connections.py

```diff
@@ -125,13 +145,33 @@ def tcp_thd_conn(port, host='localhost', name=None, **kwargs):  # kwargs to pass
                               port=port, host=host, **kwargs)  # TODO: add name
         return io_conn
 
+    def sshshell_thd_conn(host=None, port=None, username=None, login=None, password=None, name=None,
+                          reuse_ssh_of_shell=None, **kwargs):
+        mlr_conn = mlr_conn_utf8_with_clean_vt100(moler_conn_class, name=name)
+        if reuse_ssh_of_shell:
+            assert (host is None) and (port is None) and (username is None) and (login is None) and (password is None)
```

Comment:
Asserts can be disabled in run time. So the conditions will not be checked.

Reply:
you are right. done